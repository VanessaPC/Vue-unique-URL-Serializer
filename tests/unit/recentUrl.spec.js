import { render } from "@testing-library/vue";
import "@testing-library/jest-dom/extend-expect";

import RecentUrl from "@/components/recent-urls/index.vue";

RecentUrl.methods.init = jest.fn();

describe("RecentUrl component", () => {
  beforeEach(() => {
    RecentUrl.methods.init.mockImplementation(function() {
      this._data.urls = [{ _id: "1234", url: "asd" }];
    });
  });

  it("should render UserInputURL", () => {
    const { container } = render(RecentUrl);

    expect(container).toMatchSnapshot();
  });
});
