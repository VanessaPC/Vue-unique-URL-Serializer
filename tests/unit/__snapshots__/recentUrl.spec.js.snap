// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`RecentUrl component should render UserInputURL 1`] = `
<div>
  <div
    class="kLLGnf"
  >
    <h2>
      
    Recently shortened URL's
  
    </h2>
    
  []
   
    <p
      data-testid="url-empty-container"
      style="display: none;"
    >
      
    There are no recently shortned URL's
  
    </p>
  </div>
</div>
`;
